package.json

```diff
@@ -40,7 +40,7 @@
     "dev": "gulp dev",
     "lint": "npm run eslint && npm run stylelint",
     "eslint": "eslint --report-unused-disable-directives --ignore-path .gitignore .",
-    "stylelint": "stylelint src/css/ --rd --risd",
+    "stylelint": "stylelint src/css/**/*.scss --rd --risd",
```

Comment:
Are you sure this is needed? BTW when using wildcards here you need to double quote them.

Reply:
Originally I had the fonts in the CSS folder, but it wasn't the right place for them. I'll revert this change. Good catch!